.github/workflows/test.yml

```diff
@@ -77,5 +79,9 @@ jobs:
       - run: go mod download
       - env:
           TF_ACC: "1"
-        run: go test -v -cover ./internal/provider/
+          INFISICAL_HOST: ${{ secrets.INFISICAL_HOST }}
+          INFISICAL_UNIVERSAL_AUTH_CLIENT_ID: ${{ secrets.INFISICAL_UNIVERSAL_AUTH_CLIENT_ID }}
+          INFISICAL_UNIVERSAL_AUTH_CLIENT_SECRET: ${{ secrets.INFISICAL_UNIVERSAL_AUTH_CLIENT_SECRET }}
+          INFISICAL_TEST_ORG_ID: ${{ secrets.INFISICAL_TEST_ORG_ID }}
```

Comment:
Testing locally:

Makefile: 
```
testacc:
	TF_ACC=1 go test ./... -v $(TESTARGS) -timeout 120m
```

`./...` results in no tests being run. If I replace `./...` with `./internal/testacc`, it works. Are you sure `./...` works?

Reply:
Hmm let me check it out. The makefile has the same command.